hrm-service/src/contact-job/client-sqs.ts

```diff
@@ -65,7 +65,7 @@ export const publishToContactJobs = async (params: PublishToContactJobsTopicPara
   //TODO: more robust error handling/messaging
   try {
     const QueueUrl = getSsmParameter(
-      `/${process.env.NODE_ENV}/sqs/jobs/contact/queue-url-${params.jobType}`,
+      `/${process.env.NODE_ENV}/${process.env.AWS_DEFAULT_REGION}/sqs/jobs/contact/queue-url-${params.jobType}`,
```

Comment:
Sorry, not related to the PR, just noticing we are making NODE_ENV match our 'Aselo environment' in some cases. I suspect it's fine, we just need to be aware that some 3rd party libs & systems use NODE_ENV to determine some of their behaviours, so there's a small chance might have cause to set it independently of our Aselo Environment in the future, but I suspect it's straightforward enough to change in the future if it ever comes up. Just thought it was worth mentioning

Reply:
Totally! It was already being set to `development'/`staging`/`production` for the ECS container via the `.env` files and is a pattern that I've used before as well so I continued with it. I've also run into issues before as well, but mostly if I've wanted to use `test` for something other than tests. But definitely keeping the possibility in mind.